ClientRuntimes/NodeJS/lib/validate.js

```diff
@@ -52,19 +52,18 @@ exports.isValidUuid = function(uuid, callback) {
 };
 
 /**
-* Creates a anonymous function that check if a given key is base 64 encoded.
+* Validates if the given string is base64 encoded
 *
 * @param {string} key The key to validate.
-* @return {function}
+* @return {bool} 
 */
 exports.isBase64Encoded = function (key) {
```

Comment:
please rename the "key" to be more general naming as we are not deal with any dictionary, so that we have to use "key". The "str" would be enough.